mostlyai/engine/_tabular/probability.py

```diff
@@ -348,30 +411,30 @@ def predict_proba(
         # Compute total number of probability values
         total_cardinality = 1
         for target_col in target_columns:
-            col_stats = tgt_stats["columns"][target_col]
+            col_stats = ctx.tgt_stats["columns"][target_col]
             target_cardinality = sum(col_stats["cardinalities"].values())
             total_cardinality *= target_cardinality
 
-        if total_cardinality > 100:
-            _LOG.warning(
-                f"Computing joint probabilities for {len(target_columns)} targets "
-                f"results in {total_cardinality:,} total probability values per sample. "
-                f"Computation complexity grows exponentially with the number of targets. "
-                f"Consider computing probabilities for targets separately if this takes too long."
-            )
+        _LOG.warning(
+            f"Computing joint probabilities for {len(target_columns)} targets "
+            f"results in {total_cardinality:,} total probability values per sample. "
+            f"Computation complexity grows exponentially with the number of targets. "
+            f"Consider computing probabilities for targets separately if this takes too long."
+        )
 
     # Initialize with first target: P(col1)
     first_target_df = _generate_marginal_probs(
-        model=model,
+        model=ctx.model,
         seed_encoded=seed_encoded,
         target_column=target_columns[0],
-        tgt_stats=tgt_stats,
-        tgt_cardinalities=tgt_cardinalities,
-        all_columns=all_columns,
-        device=device,
+        tgt_stats=ctx.tgt_stats,
+        tgt_cardinalities=ctx.tgt_cardinalities,
```

Comment:
`ctx.tgt...` creates a dissonance. How about choosing a different and more unique name across our code? E.g. `ModelContext` -> `ModelBundle` or `ModelArtifacts`. Then, `bundle.tgt_stats` would be immediately clear.